tests/integration/istio_io/main_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istio_io
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+)
+
+var (
+	ist istio.Instance
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Does it make sense to assume the same configuration for everything? For instance, some tests expect Istio to have been deployed with mTLS enabled and others with it disabled, not sure if there's other conflicting environmental assumptions.

Reply:
If they require a different configuration, they will have a subdirectory. If not, they get put into the same package so that we only have to start/stop the system once. Make sense?